README.md

```diff
@@ -6,11 +6,19 @@ A GraphQL Client for .NET Standard over HTTP.
 
 ## Specification:
 The Library will try to follow the following standards and documents:
-[GraphQL Specification](https://facebook.github.io/graphql/June2018)
-[GraphQL HomePage](http://graphql.org/learn)
+
+* [GraphQL Specification](https://facebook.github.io/graphql/June2018)
```

Comment:
This link redirects to http://spec.graphql.org/ for a long time

```suggestion
* [GraphQL Specification](https://spec.graphql.org/June2018/)
```

Reply:
Good suggestion; accepted!